src/uct/ib/rdmacm/rdmacm_cm.h

```diff
@@ -50,6 +56,36 @@ typedef struct uct_rdmacm_cm_config {
     double          timeout;
 } uct_rdmacm_cm_config_t;
 
+struct uct_rdmacm_cm_reserved_qpn_blk {
```

Comment:
- pls document the struct and its fields
- 2 space lines between structs in H file

Reply:
@yosefe Fixed.